app/views/application/index.html

```diff
@@ -114,6 +114,22 @@ <h2 class="govuk-heading-m govuk-!-font-size-27">Evidence of expertise</h2>
           {% endif %}
         </ul>
       </section>
+
+      <section class="govuk-!-margin-bottom-8">
+        <h2 class="govuk-heading-m govuk-!-font-size-27">Competencies/Soft skills</h2>
```

Comment:
I think we could have this whole section within the one above. So move both the `{{ appTaskListItem({  ` to above the `{% endif %} `on line 114